libs/langchain/langchain/llms/databricks.py

```diff
@@ -24,14 +24,22 @@ class _DatabricksClientBase(BaseModel, ABC):
     api_url: str
     api_token: str
 
-    def post_raw(self, request: Any) -> Any:
+    def request(self, method: str, url: str, request: Any) -> Any:
         headers = {"Authorization": f"Bearer {self.api_token}"}
-        response = requests.post(self.api_url, headers=headers, json=request)
+        response = requests.request(
+            method=method, url=url, headers=headers, json=request
+        )
         # TODO: error handling and automatic retries
         if not response.ok:
             raise ValueError(f"HTTP {response.status_code} error: {response.text}")
         return response.json()
 
+    def _get(self, url: str) -> Any:
+        return self.request("GET", url, None)
+
+    def _post(self, url: str, request: Any) -> Any:
+        return self.request("POST", url, request)
```

Comment:
@harupy Can we update this class to use Databricks Deployment client from MLflow? The current implementation of this is probably really flaky because it doesn't handle transient errors, has no retry with exponential backoff

We should also ping the author of this module and make sure they're okay with us introducing MLflow as a dependency (should be fine...)

Reply:
@mengxr

Reply:
@harupy Can we remove `class _DatabricksClientBase(BaseModel, ABC)`?

Reply:
I'd remove it if we didn't have `_DatabricksClusterDriverProxyClient`. I want to keep `_DatabricksClientBase` to clarify what the client should implement.

Reply:
ah sorry, I missed that :)